game/scripts/vscripts/libraries/timers.lua

```diff
@@ -84,11 +89,10 @@ end
 
 function Timers:start()
 	Timers = self
-	self.timers = {}
+	self.timers = {realTime = BinaryHeap("endTime"), gameTime = BinaryHeap("endTime")}
```

Comment:
Since I don't think there would be any other timers, could it just be `self.realTimeHeap` and `self.gameTimeHeap`?